### DISCUSSION TITLE
Motivational Speaking / Writing Topics

### DISCUSSION THREAD

**Discussion by rahulpatelio** [Mintbean/DevCollective.io#21] [DevRel]
Motivational writing: Post Bootcamp, feeling lost and what to do.

Talk about the reality of post Bootcamp. Not all get hired. What to do to keep learning and building your experience ie MintBean Learnathon. 

Pretty sure I want to write this.  :)

**Comment by monarchwadia** [comment]
Cool, seems like a pretty clear topic. Go for it! I request that you portray bootcamps in an unbiased and objective light, since Mintbean is supportive of what bootcamps do in general.
Anything you need from us?

**Reply by rahulpatelio** [reply]
Just working on a draft at the moment. Of course, this article isn't airing out any frustrations about boot camps. It's more a "what's next after Bootcamp realistically" approach.

**Reply by monarchwadia** [reply]
That sounds great! Will you be publishing it on Medium? That way we can add it to the Mintbean publication.

**Comment by dinaporia** [comment]
I could definitely use an article like this right about now! I'm happy to proofread / comment before publishing if you'd like.

**Reply by rahulpatelio** [reply]
I would love that. I'm working on a draft as we speak. Aiming to publish by end of next week.
